sagemaker-python-sdk/tensorflow_script_mode_mnist/tensorflow_distributed_mnist.ipynb

```diff
@@ -0,0 +1,224 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# MNIST distributed training  \n",
```

Comment:
I'd advertise in the title that this is using TF script mode

also nit: remove the trailing whitespace